src/meetingsession/MeetingSessionStatusCode.ts

```diff
@@ -120,14 +120,19 @@ export enum MeetingSessionStatusCode {
   IncompatibleSDP = 21,
 
   /**
-   * This can happen when you attempt to join a meeting which has ended
+   * This can happen when you attempt to join a meeting which has ended or attendee got removed
    */
   TURNCredentialsForbidden = 22,
```

Comment:
Is there any difference between "TURNCredentialsForbidden" and "TURNMeetingEnded" when attempting to join an ended meeting ?

Reply:
Existing endpoint behavior is incorrect since it always return 403. I plan to change that, but client-side fix must go first.